server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -181,7 +185,36 @@ private FormTag buildProgramForm(
             .setValue(DisplayMode.SELECT_TI.getValue())
             .setChecked(displayMode.equals(DisplayMode.SELECT_TI.getValue()))
             .getRadioTag(),
-        showTiSelectionList(selectedTi, displayMode.equals(DisplayMode.SELECT_TI.getValue())),
+        showTiSelectionList(selectedTi, displayMode.equals(DisplayMode.SELECT_TI.getValue())));
+
+    formTag.with(
+        // TODO(#2618): Consider using helpers for grouping related radio controls.
+        fieldset()
+            .with(
+                legend("Program eligibility gating")
+                    .withClass(BaseStyles.INPUT_LABEL)
+                    .with(ViewUtils.requiredQuestionIndicator())
+                    .with(p("(Not applicable if this program is the pre-screener)")),
+                FieldWithLabel.radio()
+                    .setFieldName(ELIGIBILITY_IS_GATING_FIELD_NAME)
+                    .setAriaRequired(true)
+                    .setLabelText(
+                        "Eligibility criteria blocks submission (applicants must meet all"
```

Comment:
suggestion (blocking): My brain had a hard time parsing the meaning of this sentence. That could just be because eligibility stuff is a brain twister and takes some extra thinking, but it would be great to get the sign off on this from design and/or product. (update: we chatted about this in Slack and Sijia will take a look when they're back)

Reply:
Hah, I tested this text on https://hemingwayapp.com/ and it got a reading level of "post-graduate" 👎 Re-writing to:
- Residents can only submit applications if they meet all the eligibility requirements
- Residents can submit applications even if they don't meet eligibility requirements

takes it down to grades 13/14, which still isn't great but better.

Removing "eligibility requirements" takes it down to grade 7, so I think that phrase will always spike the rating, but that's also the phrasing we use everywhere. I'll run the wording by Sijia!